### ISSUE TITLE
Stuck at attempting to connect 

### ISSUE THREAD

**Issue by osamahesham357** [wuphilipp/gello_software#18]
The code was working fine, then it started to stuck at attempting to connect, whatever I do it stuck  there.
![Screenshot from 2024-07-31 09-45-49](https://github.com/user-attachments/assets/688ce802-e1a9-4070-8abb-342c7e5ffa84)

**Comment by wuphilipp** [comment]
Humm unfortunatly I have not encountered this issue, so don't have any concreate advice. Do you have more detailed diagnostics? Have you tried restarting the system or the dynamixels?

**Comment by OH357** [comment]
Yes i tried restarting, but sometimes it works, i think the issue is to get
the connection up to the simulation or the real robot, if the simulation or
real robot has an error and didn't run correctly, this happens.
I also have another question any way to run both simulation and real
robotat same time?
OH

**Comment by fireman5379** [comment]
Same! have you solved it? @OH357

**Comment by OH357** [comment]
It was an IP issue while connecting to the real robot, I found it weird as it kept mentioning it's a usb serial port issue but after debugging it was an issue with the IP.
